bug-triage/triage-workflow.md

```diff
@@ -17,255 +47,101 @@ The first step of triage is to confirm that the issue is valid and filled in pro
+  be relevant for the current development cycle.
+* *The current development version*: Should be assigned to issues that are [release blockers](/bug-triage/release-blockers.md), or otherwise prioritized
+  for the current version.
+* *The next release version*: When we enter feature freeze we usually create a new milestone used for PRs that are approved but won't make it into
+  the current release, this milestone is not used for issues.
+* *Older Godot 4 versions*: This is used for issues that are only relevant for this specific version (or older versions), but not any newer version.
+  An example of this would be an issue that was solved in `4.5` as part of an enhancement, but that enhancement cannot be cherry-picked for `4.4` and
+  a separate issue is necessary to track the specific solution for `4.4` (and older, if relevant). For such issues it can also be relevant to add “[4.4]” at
+  the beginning of the issue title to help clarify it is specific to this version.
 
-### GDExtension
+For Godot 3:
+* *`3.x`*: For the `3.x` branch in general. Used for issues that are only relevant for the `3.x` version, and occurs on the current development version of `3.x`.
+  For these issues it can also help to add “[3.x]” at the beginning of the issue title to help identifying the issue.
+* Other Godot 3 milestones work the same way as for the `master` branch, except we do not track release blockers for `3.x`.
 
-Tracked in:
-* [GDExtension Issue Triage](https://github.com/orgs/godotengine/projects/81/views/1)
+## Testing an Issue
 
-RC Channel:
-* [#gdextension](https://chat.godotengine.org/channel/gdextension)
-
-### GDScript
-
-Tracked in:
-* [GDScript Issue Triage](https://github.com/orgs/godotengine/projects/79)
-
-RC channel:
-* [#gdscript](https://chat.godotengine.org/channel/gdscript)
-
-### GUI
-
-Tracked in:
-* [GUI Issue Triage](https://github.com/orgs/godotengine/projects/100)
-
-RC channel:
-* [#gui](https://chat.godotengine.org/channel/gui)
-
-### Navigation
-
-Tracked in:
-* [Navigation Issue Triage](https://github.com/orgs/godotengine/projects/103)
-
-RC channel:
-* [#navigation](https://chat.godotengine.org/channel/navigation)
-
-### .NET / Mono
-
-Tracked in:
-* [.NET Issue Triage](https://github.com/orgs/godotengine/projects/83) (Internal tracker)
-
-RC channel:
-* [#dotnet](https://chat.godotengine.org/channel/dotnet)
-
-### Network
-
-Tracked in:
-* [Network Issue Triage](https://github.com/orgs/godotengine/projects/96)
-
-RC channel:
-* [#networking](https://chat.godotengine.org/channel/networking)
-
-### Particles
+A valid MRP is a *minimal* project that reproduces a bug. This means that it is no larger than it needs to be, it also has to be a project, not an exported executable.
+Do _not_ run executable projects added to a bug report, they are not valid MRPs as an MRP needs to be something that can be evaluated in detail, and be tweaked if needed,
+and more importantly they are untrusted files.
```

Comment:
suggestion: Might be worth warning about Godot projects too. Tool scripts can essentially run on open and could be used to do damage, so we probably want to suggest using a code editor that doesn't run the code or something like VSCode's restricted mode. I always tend to use vanilla vim to check the script files in an MRP first, and if it has more than a couple then I won't open it at all. Only once I'm happy it is safe do I VSCode + Godot Editor to test it out.